docs/modules/session-explorer/pages/manage-sessions.adoc

```diff
@@ -7,27 +7,21 @@ Learn how to manage sessions so you can rename a session, change the session des
 
 In Kobiton, select *Sessions*.
 
-image:select-sessions-closeup.png[width=1000,alt="Select Sessions tab"]
+image:select-sessions-closeup.png[width=150,alt="Select Sessions tab"]
 
 xref:session-explorer:search-for-a-session.adoc[Search for a session], then select the session ID to open the session overview.
 
 image:search-session-id-closeup.png[width=1000,alt="Search for a session ID"]
 
-From the Session Overview window, users can view below mentioned information.
+From the **Session Overview** window, you can view session info, device info, logs, and more.
```

Comment:
let's remove window as we usually use page if it's needed.
I'd prefer to say - "From Session Overview, you..." and add video to the list.